Tools/cases_generator/cwriter.py

```diff
@@ -0,0 +1,89 @@
+
+from lexer import Token
+
+class CWriter:
+    'A writer that understands how to format C code.'
+
+    def __init__(self, out, indent = 0):
```

Comment:
I'm finding MyPy to be close to unusable.
It will not do the most basic inference. Having to annotate every function that does not have a `return` with `-> None` is just extra work and noise.

Reply:
Let’s talk options when I am back.

Reply:
I've worked around the MyPy issues. Mainly renaming variables, adding `__all__`, and adding a bunch of ` -> None`s.

Not being able to trust the output of MyPy undermines the value of the one or two errors it did find.

Reply:
With the correct (but non-obvious) flags set, it works as expected.